src/System.Numerics.Vectors/src/Configurations.props

```diff
@@ -1,8 +1,8 @@
-﻿<Project>
+﻿<Project DefaultTargets="Build">
```

Comment:
This `DefaultTargets="Build"` entry is not needed, please remove.

Reply:
Fixed.